tests/integration/lambdas/lambda_s3_integration.js

```diff
@@ -0,0 +1,44 @@
+exports.handler = async (event, context, callback) => {
+    const {
+        S3Client,
+        PutObjectCommand,
+    } = require("@aws-sdk/client-s3");
+    const {
+        getSignedUrl
+    } = require('@aws-sdk/s3-request-presigner');
+
+    const CREDENTIALS = {
+        secretAccessKey: 'test',
+        accessKeyId: 'test',
+    };
+
+    const ENDPOINT = {
+        path: '',
+        hostname: 's3.localhost.localstack.cloud:4566',
+        protocol: 'http',
+    };
+
+    const BUCKET_NAME = process.env.AWS_LAMBDA_FUNCTION_NAME;
+
+    const s3 = new S3Client({
+        endpoint: ENDPOINT,
+        region: 'us-east-1',
```

Comment:
nitpick: Should be good for now, but perhaps we can pass the region (or read it from the environment) here in the future.